enlearn.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   ],
   extends: [
     "eslint:recommended",
+    "plugin:jest/recommended",
```

Comment:
Make sure this plugin is added both as a peer dependency in the package.json and the install script listed in our readme is updated accordingly, so it's easier for folks to get things installed when setting up new projects.